core/modules/ccontrol/UserQuerySelect.h

```diff
@@ -0,0 +1,147 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_QSERV_CCONTROL_USERQUERYSELECT_H
+#define LSST_QSERV_CCONTROL_USERQUERYSELECT_H
+/**
+  * @file
+  *
+  * @brief Umbrella container for user query state
+   *
+  * @author Daniel L. Wang, SLAC
```

Comment:
Yes, git unfortunately tends to think that file is new if you just copy it and make few simple changes to it. But it still thinks that .cc files are related (UserQuery.cc → UserQuerySelect.cc). I don't think there is an easy workaround for that besides committing right after renaming.